mne/commands/tests/test_commands.py

```diff
@@ -177,4 +178,34 @@ def test_surf2bem():
     check_usage(mne_surf2bem)
 
 
+@slow_test
+@requires_freesurfer
+@sample.requires_sample_data
+def test_watershed_bem():
+    """Test mne watershed bem"""
+    check_usage(mne_watershed_bem)
+    orig_subject_dir = os.getenv('SUBJECTS_DIR', None)
+    orig_subject = os.getenv('SUBJECT', None)
```

Comment:
You get these values, but you never actually set `os.environ[...]`. So please add that bit.

Also, once you do that, please put a `try/except/finally` to set the `environ` back to normal no matter what errors get thrown.